lib/cloudregister/registerutils.py

```diff
@@ -503,6 +503,23 @@ def get_credentials(credentials_file):
     return (username, password)
 
 
+# ----------------------------------------------------------------------------
+def refresh_registry_credentials():
+    """Refresh registry credentials."""
+    requests.packages.urllib3.disable_warnings() # to silence InsecureRequestWarning
```

Comment:
We shouldn't have an insecure request from a client.

Reply:
Agree, but the fix for that should be in a different PR, IMO